pkgs/development/libraries/kerberos/heimdal.nix

```diff
@@ -136,6 +140,10 @@ stdenv.mkDerivation {
     export USER=nix-builder
   '';
 
+  postConfigure = ''
+    echo "#define HAVE_DB_185_H 1" >> include/config.h
```

Comment:
Would be nice to have a comment why we added this, so that we remember in the future.

Reply:
I think there's a somewhat better way to fix it; we can patch their autoconf files to define this macro. I will send an update and post a fix for upstream.